packages/vite/src/node/utils.ts

```diff
@@ -58,8 +58,10 @@ interface DebuggerOptions {
   onlyWhenFocused?: boolean | string
 }
 
+type ViteDebugScope = `vite:${string}`
```

Comment:
If we do this, I would suggest to also `export` it.
Just in case someone external also want to depend on it.

Reply:
done